legal_db/models.py

```diff
@@ -1,11 +1,71 @@
+from django.contrib.postgres.fields import ArrayField
 from django.db import models
+from django_countries.fields import CountryField
 
 
-class License(models.Model):
-    code = models.CharField(max_length=12)
-    title = models.CharField(max_length=50)
-    version = models.CharField(max_length=25, blank=True, null=True)
+class LegalResource(models.Model):
+    license = models.CharField(max_length=25, blank=True, null=True)
```

Comment:
do we need three versions for licenses (license, version, is_ported)? we discussed a single string field.

I'll defer to @TimidRobot here.